src/test/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/input/Generators.scala

```diff
@@ -55,7 +57,8 @@ object Generators {
         classes          = classes,
         autocomplete     = autocomplete,
         pattern          = pattern,
-        attributes       = attributes
+        attributes       = attributes,
+        spellcheck       = arbSpellcheck
```

Comment:
You can correct arbSpellcheck back to spellcheck now, I should have done that before checking in, apologies.